egui/src/response.rs

```diff
@@ -277,6 +277,27 @@ impl Response {
         self.drag_released
     }
 
+    /// Mouse dragged, nevertheless it's stuck on screen's edge or not.
+    /// The idea of this method is to overcome limitations of computing delta from `old-new`,
+    /// which does not work when mouse is stuck on screen corner.
+    /// On platform where it is not supported in fallbacks to `pointer.delta()`
+    pub fn motion_delta(&self) -> Vec2 {
+        #[cfg(target_family = "wasm")]
+        return self.ctx.input().pointer.delta();
+
+        #[cfg(not(target_family = "wasm"))]
+        {
+            let res = self.ctx.input().pointer.motion();
+
+            if res == Vec2::ZERO {
+                // in a case of touchscreen or whatever
+                self.ctx.input().pointer.delta()
```

Comment:
maybe touch screens should set `motion` too?

Better yet: `RawInput` could have `motion: Option<Vec2>` and then `InputState::begin_frame` would use it if set, or else fall back to the pointer delta. That way it will work correctly for all existing backends.